tensorflow/contrib/rnn/python/kernel_tests/core_rnn_test.py

```diff
@@ -2251,8 +2250,7 @@ def _assert_in(op_str, in_stats, out_stats):
     _assert_in("TensorArray", cpu_stats, gpu_stats)
 
   def testInputOnGPUCellNotDeclared(self):
-    # Currently not implemented for OpenCL
-    if not test.is_gpu_available() or test_util.is_sycl_enabled():
```

Comment:
Can that be done as a separate commit? This already has been reverted in the opencl_improvements branch and will cause merge conflict